### PR TITLE
contacts

### PR DESCRIPTION
**Pull request by quangchien0806**

### CODE REVIEW

**Comment by quangchien0806** [comment]
[Add Contacts Section](https://trello.com/c/Vodpazzl/62-add-contacts-section)

**Comment by quangchien0806** [comment]
[Add Contacts Section](https://trello.com/c/Vodpazzl/62-add-contacts-section)

File: src/components/Chats/ChatLists/index.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import React from "react";
 import Scrollbars from "react-custom-scrollbars-2";
 import Avatar from "../../Avatar";
-import { dataChat } from "../dataChat";
 
 import "./style.scss";
+import { useSelector } from "react-redux";
```
Comment (phongnguyenhh1996) [inline src/components/Chats/ChatLists/index.js]: lib import move to top, style import move to bottom

File: src/components/Chats/index.js [hunk 2]
```diff
@@ -1,20 +1,31 @@
 import React from "react";
 import { CiSearch } from "react-icons/ci";
 import ScrollContainer from "react-indiana-drag-scroll";
+import { onSearch, selectChatList } from "../../features/chats/chat";
 import Avatar from "../Avatar";
 import ChatList from "./ChatLists";
-import { dataChat } from "./dataChat";
+
 import "./style.scss";
+import { useSelector, useDispatch } from "react-redux";
```
Comment (phongnguyenhh1996) [inline src/components/Chats/index.js:11]: lib import move to top, style import move to bottom

**Review by phongnguyenhh1996 — APPROVED** [review]

**Review by phongnguyenhh1996 — APPROVED** [review]

**Pull request merged**